Library/ViewModels/RewardAddOnSelectionViewModel.swift

```diff
@@ -330,13 +330,34 @@ public final class RewardAddOnSelectionViewModel: RewardAddOnSelectionViewModelT
       )
     }
 
-    Signal.zip(project, baseReward, refTag)
-      .takeWhen(self.continueButtonTappedProperty.signal)
-      .observeForUI()
-      .observeValues { project, baseReward, refTag in
+    // Send updated checkout data with add-ons continue event
+    Signal.combineLatest(
```

Comment:
Looks like the add ons total count and unique count are showing up but I'm noticing another discrepancy you should look into.

I'm using the project `The Tarot Restless` and adding the following:
The $30 reward (2nd to 3rd Edition Booster Set)
2 x $15 add on (11" x 17" poster)
1 x $35 add on (18" x 24" poster)

The total in the pledge view after adding shipping is $105 but we are sending:
checkout_amount : "100.00",
checkout_amount_total_usd : 100,


My hunch is that shipping for the reward is not being accounted for.

Reply:
Okay, I'll check that out